lib/rodauth/features/login.rb

```diff
@@ -86,8 +89,12 @@ def login(auth_type)
         yield if block_given?
         after_login
       end
+      login_response(saved_login_redirect)
+    end
+
+    def login_response(redirect_url)
```

Comment:
Seems odd to have `login_response` take an argument when the other `response` methods don't.  Maybe add a private attr_reader for `saved_login_redirect`, and call that inside `login_response`?

Reply:
done